cmd/operator/deploy/operator/12-alertmanager.yaml

```diff
@@ -60,7 +60,7 @@ spec:
     spec:
       containers:
       - name: alertmanager
-        image: prom/alertmanager:latest
+        image: gcr.io/gke-release-staging/prometheus-engine/alertmanager:v0.0.0-gmp.1-gke.0
```

Comment:
obviously we won't want to release v0.0.0 (or staging), just opening this to show the change needed